pkg/commands/check.go

```diff
@@ -201,3 +238,57 @@ func normalize(line string) string {
 func denormalize(line string) string {
 	return strings.ReplaceAll(line, "YYYY", fmt.Sprint(time.Now().Year()))
 }
+
+func (co *checkOptions) fixMissingBoilerplate(path string) error {
+	content, err := ioutil.ReadFile(path)
+	if err != nil {
+		return err
+	}
+
+	boilerplate := denormalize(strings.Join(co.boilerplateLines, "\n")) + "\n"
+	newContent := boilerplate + string(content)
+
+	return ioutil.WriteFile(path, []byte(newContent), 0644)
+}
```

Comment:
This makes sense to me and feels like the most common in practice.  I worry the other two are too clever.